test/LiquidityManager/invest.test.ts

```diff
@@ -62,15 +74,34 @@ describe.only('LiquidityManager#invest', () => {
         )
     }
 
-    async function sortTokens(tokenA: TestERC20, tokenB: TestERC20) {
-        const [addressTokenA, addressTokenB] = await Promise.all([
-            unwrapAddressLike(tokenA),
-            unwrapAddressLike(tokenB),
+    async function sortTokensAndPrices(
+        token0: TestERC20,
+        token0Price: bigint,
+        token1: TestERC20,
+        token1Price: bigint,
+    ) {
+        const [addressToken0, addressToken1] = await Promise.all([
+            unwrapAddressLike(token0),
+            unwrapAddressLike(token1),
         ])
 
-        return addressTokenA.toLowerCase() > addressTokenB.toLowerCase()
-            ? { token0: tokenB, token1: tokenA }
-            : { token0: tokenA, token1: tokenB }
+        if (addressToken0.toLowerCase() > addressToken1.toLowerCase()) {
+            [
+                token0,
+                token0Price,
+                token1,
+                token1Price,
+            ] = [token1, token1Price, token0, token0Price]
+        }
+
+        return {
+            token0,
+            token0Price,
```

Comment:
i dont think we should be using prices in `bigint` since we will be reusing a lot of code in the front end and prices will always have important decimals